projects/netgrif-components/src/lib/side-menu/content-components/admin-impersonate/admin-impersonate-list/admin-impersonate-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component} from '@angular/core';
+import {AbstractUserAssignListComponent, UserListService} from '@netgrif/components-core';
+
+/**
+ * Is responsible for displaying, filtering, loading and selecting users.
+ */
+@Component({
+    selector: 'nc-admin-impersonate-list',
+    templateUrl: './admin-impersonate-list.component.html',
+    styleUrls: ['./admin-impersonate-list.component.scss']
+})
+export class AdminImpersonateListComponent extends AbstractUserAssignListComponent {
+
+    constructor(protected _userListService: UserListService) {
```

Comment:
if passed as argument to super-constructor, then `protected` should be dropped

Reply:
done